src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -666,6 +666,13 @@
      static_field(VMRegImpl,                   regName[0],                                    const char*)                           \
      static_field(VMRegImpl,                   stack0,                                        VMReg)                                 \
                                                                                                                                      \
+  /**************/                                                                                                                   \
+  /* CI (NOTE: these fields should not be removed, they can be used by external tools) */                                            \
```

Comment:
Can you also clarify what "external tools" are in the comment so that when/if those external tools stop using them, we can re-evaluate?